apps/emqx/src/emqx_limiter/etc/emqx_limiter.conf

```diff
@@ -40,10 +31,16 @@ limiter {
   }
 
   message_routing {
-    global.rate = infinity
-    zone.default.rate = infinity
     bucket.default {
-      zone = default
+      aggregated.rate = infinity
+      aggregated.capacity = infinity
+      per_client.rate = infinity
+      per_client.capacity = infinity
+    }
+  }
+
+  shared {
```

Comment:
We need readable and detailed comments for each type of limiter name

Reply:
yes i will add it